clients/src/main/java/org/apache/kafka/clients/admin/ListConsumerGroupOffsetsOptions.java

```diff
@@ -44,10 +45,21 @@ public ListConsumerGroupOffsetsOptions topicPartitions(List<TopicPartition> topi
         return this;
     }
 
+    /**
+     * Sets an optional requireStable flag.
+     */
+    public void requireStable(final boolean requireStable) {
+        this.requireStable = requireStable;
+    }
+
     /**
      * Returns a list of topic partitions to add as part of the result.
      */
     public List<TopicPartition> topicPartitions() {
         return topicPartitions;
```

Comment:
Yeah I think I can file a separate JIRA for it since a lot of the admin options have this open question... also it may be concerned as a backward compatibility issue, so it's better separating it from this PR.